<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeacherForm</class>
 <widget class="QWidget" name="TeacherForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1700</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1700</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1700</width>
    <height>900</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Teacher</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>uit.ico</normaloff>uit.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white
</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>10</width>
     <height>841</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #57C4C6; border-style: outset; border-width: 2px; border-radius: 5px; border-color: white; padding: 4px; color:white</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupboxProfile">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>291</width>
     <height>371</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #296091; border-color: white;</string>
   </property>
   <property name="title">
    <string>PROFILE</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QLabel" name="labelName">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>231</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: red
</string>
    </property>
    <property name="text">
     <string>NGUYỄN VĂN A</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPhone">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>221</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0123456789</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>31</width>
      <height>301</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>IconProfile.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="labelMail">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>290</y>
      <width>211</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>anv@uit.edu.vn</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFaculty">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>MMT&amp;TT</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSex">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nam</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelIDTeacher">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>MSGV</string>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>labelName</zorder>
   <zorder>labelPhone</zorder>
   <zorder>labelMail</zorder>
   <zorder>labelFaculty</zorder>
   <zorder>labelSex</zorder>
   <zorder>labelIDTeacher</zorder>
  </widget>
  <widget class="QPushButton" name="buttonLogout">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>640</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(18,153,175); border-style: outset; border-width: 2px; border-radius: 15px; border-color: white; padding: 4px; color:white</string>
   </property>
   <property name="text">
    <string>LOG OUT</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>740</y>
     <width>331</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>decorate.png</pixmap>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineSearch">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>40</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>12</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white; border-style: solid; border-width: 2px; border-radius: 10px; border-color: rgb(18,153,175); padding: 4px; color: #474747;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Tìm kiếm</string>
   </property>
  </widget>
  <widget class="QLabel" name="buttonSearch">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>40</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>lookup.png</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>211</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: outset; border-width: 2px; border-radius: 5px; border-color: white; padding: 4px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>image.png</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="selectYear">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>170</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:solid;
border-width: 1px; 
border-color:  black;
color: black</string>
   </property>
   <item>
    <property name="text">
     <string>2020-2021</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2019-2020</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2018-2019</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="selectSemester">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>120</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:solid;
border-width: 1px; 
border-color:  black;
color: black</string>
   </property>
   <item>
    <property name="text">
     <string>Học kỳ 1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Học kỳ 2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Học kỳ hè</string>
    </property>
   </item>
  </widget>
  <widget class="QGroupBox" name="groupListClass">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>250</y>
     <width>261</width>
     <height>371</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(18,153,175);
</string>
   </property>
   <property name="title">
    <string>DANH SÁCH LỚP</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QListWidget" name="listClass">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>231</width>
      <height>301</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white; border-style: solid; border-width: 0px; color: #474747;</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <item>
     <property name="text">
      <string>NT219.L21.ANTT</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>NT203.ANTT</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>IT007.L21</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QTableWidget" name="tableClass">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>160</y>
     <width>1001</width>
     <height>691</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: white; border-color: red

</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <column>
    <property name="text">
     <string>MSSV</string>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Họ và tên</string>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Giới tính</string>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Điểm QT</string>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Điểm GK</string>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Điểm TH</string>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Điểm CK</string>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Điểm TB</string>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Ghi chú</string>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>120</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black
</string>
   </property>
   <property name="text">
    <string>ID lớp:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelIDClass">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>120</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: red
</string>
   </property>
   <property name="text">
    <string>NT219.L21.ANTT</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelNameSubject">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>120</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: red
</string>
   </property>
   <property name="text">
    <string>Mật mã học</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>120</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black
</string>
   </property>
   <property name="text">
    <string>Môn học:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelNumber">
   <property name="geometry">
    <rect>
     <x>1610</x>
     <y>120</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: red
</string>
   </property>
   <property name="text">
    <string>70</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>1520</x>
     <y>120</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black
</string>
   </property>
   <property name="text">
    <string>Sĩ số:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>50</y>
     <width>401</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(41, 96, 145)
</string>
   </property>
   <property name="text">
    <string>BẢNG ĐIỂM SINH VIÊN</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSave">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>640</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 85, 127); border-style: outset; border-width: 2px; border-radius: 15px; border-color: white; padding: 4px; color:white</string>
   </property>
   <property name="text">
    <string>LƯU ĐIỂM</string>
   </property>
  </widget>
  <zorder>buttonSearch</zorder>
  <zorder>lineEdit</zorder>
  <zorder>groupboxProfile</zorder>
  <zorder>buttonLogout</zorder>
  <zorder>label_3</zorder>
  <zorder>lineSearch</zorder>
  <zorder>label</zorder>
  <zorder>selectYear</zorder>
  <zorder>selectSemester</zorder>
  <zorder>groupListClass</zorder>
  <zorder>tableClass</zorder>
  <zorder>label_4</zorder>
  <zorder>labelIDClass</zorder>
  <zorder>labelNameSubject</zorder>
  <zorder>label_5</zorder>
  <zorder>labelNumber</zorder>
  <zorder>label_7</zorder>
  <zorder>label_6</zorder>
  <zorder>buttonSave</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
